tldw_Server_API/app/core/LLM_Calls/providers/mlx_provider.py

```
@@ -0,0 +1,568 @@
+        self.registry = get_mlx_registry()
+
+    def capabilities(self) -> Dict[str, Any]:
+        status = self.registry.status()
+        return {
+            "dimensions_default": None,
+            "max_batch_size": status.get("config", {}).get("max_batch_size"),
+            "default_timeout_seconds": 60,
+            "supports_embeddings": status.get("supports_embeddings", False),
+        }
+
+    def embed(self, request: Dict[str, Any], *, timeout: Optional[float] = None) -> Dict[str, Any]:
```

Ruff (0.14.6)

Unused method argument: `timeout`

(ARG002)